test/e2e/framework/framework.go

```diff
@@ -176,8 +178,20 @@ func createNamespace(client kubeclientset.Interface, baseName string) (string, e
 
 func (f *frameworkWrapper) SetUpServiceDNSControllerFixture() {
 	f.framework().SetUpServiceDNSControllerFixture()
+	f.setUpNamespaceControllerFixture(nil)
 }
 
 func (f *frameworkWrapper) SetUpIngressDNSControllerFixture() {
 	f.framework().SetUpIngressDNSControllerFixture()
+	f.setUpNamespaceControllerFixture(nil)
+}
+
+func (f *frameworkWrapper) setUpNamespaceControllerFixture(typeConfig typeconfig.Interface) {
+	if TestContext.LimitedScope || (typeConfig != nil && typeConfig.GetTarget().Kind == util.NamespaceKind) {
+		return
+	}
+	// The namespace controller is required to ensure namespaces
+	// are created as needed in member clusters in advance of
+	// propagation of other namespaced types.
+	f.SetUpControllerFixture(common.NamespaceTypeConfigOrDie(NewE2ELogger()))
```

Comment:
Does this pose implications on the managed e2e testing? It seems like we also need a check for `TestContext.TestManagedFederation` as namespaces using a test managed federation don't work without a k8s namespace controller.

Reply:
Given that e2e tests run fine with managed fixture, and namespace sync controllers were being started for non-namespace sync controllers prior to this PR, it doesn't appear that there is a problem.  

Crudtester validation of namespace deletion doesn't work for managed fixture, as would be expected when the k8s namespace controller is missing.  But for managed fixture, I don't think namespaces need to be deleted since the fixture is not persistent.